Description: Fix Makefile's creation of installation directories
 The upstream makefile has a bug that prevents installation to nonexistent
 directories, such as the local ones we use for building a package.
Author: Felix Crux <felixc@felixcrux.com>
Last-Update: 2013-05-13
Forwarded: https://github.com/rwos/gti/pull/13

--- a/Makefile
+++ b/Makefile
@@ -19,8 +19,8 @@
 	gzip -9 -n -c gti.6 > gti.6.gz
 
 install: $(PROG) $(MANPAGE)
-	$(INSTALL) $(PROG) $(BINDIR)
-	$(INSTALL_DATA) $(MANPAGE) $(MANDIR)
+	$(INSTALL) $(PROG) $(BINDIR)/$(PROG)
+	$(INSTALL_DATA) $(MANPAGE) $(MANDIR)/$(MANPAGE)
 
 uninstall:
 	rm -f $(BINDIR)/$(PROG)
